Jucer2CMake/src/reprojucer.hpp

```diff
@@ -375,6 +375,18 @@ inline void writeReprojucerCMakeLists(const Arguments& args,
 
   LineWriter wLn{outputStream};
 
+  const auto writeBracket = [&wLn](const juce::String& cmakeKeyword,
```

Comment:
I'd like the `write*` lambdas to be in the order of "least-complex" to "most-complex". With the addition of `writeBracket`, it would be:
- `writeUnquoted`
- `writeBracket`
- `writeQuoted`
- `writeList`

So please move `writeBracket` right after `writeUnquoted`.

Reply:
Addressed in 22df5395c361c114111f985a2172a8ace1563de4.